service/src/main/java/uk/gov/hmcts/reform/rd/client/OrganisationApi.java

```diff
@@ -20,7 +20,7 @@
 )
 public interface OrganisationApi {
     @GetMapping("/refdata/external/v1/organisations/users")
-    OrganisationUsers findUsersInOrganisation(
+    OrganisationUsers findUsersInLoggedUserOrganisation(
```

Comment:
if you want to change the name maybe `CurrentUser` or `LoggedInUser`

Reply:
Good shout. Will do.